Makefile.am

```diff
@@ -116,7 +116,8 @@ setup.h: $(srcdir)/setup.h.in Makefile
 	     -e 's|@prefix[@]|$(prefix)|g' \
 	     -e 's|@p4includedir[@]|$(p4includedir)|g' \
 	     -e 's|@p4_14includedir[@]|$(p4_14includedir)|g' \
-	$< >$@
+	$< >$@.tmp
+	@if cmp $@ $@.tmp; then rm $@.tmp; else mv $@.tmp $@; fi
```

Comment:
That's another solution, but just to be clear: I wasn't saying you had to remove it, just that it should be documented somewhere. =)